src/ensembl/src/content/app/browser/Browser.tsx

```diff
@@ -71,37 +57,34 @@ import styles from './Browser.scss';
 import 'static/browser/browser.js';
 
 type StateProps = {
-  activeGenomeId: string;
-  activeEnsObjectId: { [genomeId: string]: string };
+  activeGenomeId: string | null;
+  activeEnsObjectId: string | null;
+  allActiveEnsObjectIds: { [genomeId: string]: string };
   browserActivated: boolean;
   browserNavOpened: boolean;
-  browserOpenState: BrowserOpenState;
   browserQueryParams: { [key: string]: string };
-  chrLocation: BrowserChrLocation;
+  chrLocation: ChrLocation | null;
+  allChrLocations: { [genomeId: string]: ChrLocation };
   drawerOpened: boolean;
-  genomeInfo: GenomeInfoData;
   genomeSelectorActive: boolean;
   trackPanelOpened: boolean;
   launchbarExpanded: boolean;
-  exampleEnsObjects: ExampleEnsObjectsData;
+  exampleEnsObjects: EnsObject[];
   committedSpecies: CommittedItem[];
 };
 
 type DispatchProps = {
   changeBrowserLocation: (
+    genomeId: string,
     chrLocation: ChrLocation,
     browserEl: HTMLDivElement
   ) => void;
   fetchEnsObject: (ensObjectId: string, genomeId: string) => void;
-  fetchEnsObjectTracks: (ensObjectId: string, genomeId: string) => void;
-  fetchExampleEnsObjects: (genomeId: string) => void;
-  fetchGenomeInfo: (genomeId: string) => void;
-  fetchGenomeTrackCategories: (genomeId: string) => void;
+  fetchGenomeData: (genomeId: string) => void;
   replace: Replace;
   toggleDrawer: (drawerOpened: boolean) => void;
-  updateBrowserActiveGenomeIdAndSave: (genomeId: string) => void;
-  updateBrowserActiveEnsObjectIdAndSave: (objectId: string) => void;
   updateBrowserNavStates: (browserNavStates: BrowserNavStates) => void;
```

Comment:
updateBrowserNavStates is unused.

Reply:
Good catch! It wasn't used before either.